SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.h

```diff
@@ -41,12 +41,12 @@ typedef NS_ENUM(NSInteger, OWSOutgoingMessageRecipientState) {
 NSString *NSStringForOutgoingMessageRecipientState(OWSOutgoingMessageRecipientState value);
 
 typedef NS_ENUM(NSInteger, TSGroupMetaMessage) {
-    TSGroupMessageUnspecified,
-    TSGroupMessageNew,
-    TSGroupMessageUpdate,
-    TSGroupMessageDeliver,
-    TSGroupMessageQuit,
-    TSGroupMessageRequestInfo,
+    TSGroupMetaMessageUnspecified,
+    TSGroupMetaMessageNew,
+    TSGroupMetaMessageUpdate,
+    TSGroupMetaMessageDeliver,
+    TSGroupMetaMessageQuit,
+    TSGroupMetaMessageRequestInfo,
```

Comment:
These enum values didn't follow our naming convention of using the full enum name as a prefix.